### DISCUSSION TITLE
Vorbelegen bzw. zurücksetzen der Felder in der rechten Leiste

### DISCUSSION THREAD

**Discussion by runeb68** [Gorthian/HeXXen1733_Custom_Character_Sheet#37] [Ideas]
Moin

Meine Mitspieler und ich möchten die Felder rechts auf dem Bogen für die aktuelle INI, Coups, LeP, Ideen, ... gerne einfach zurücksetzen können.

Ich habe mir überlegt da wo die berechneten Maximalwerte auf den ersten beiden Reitern stehen jeweils rechts davon ein ▶️ Symbol (Pfeil nach Rechts im Quadrat) als Button einzufügen mit dem man die Felder für die aktuelle INI, Coups, LeP, Ideen, ... auf den jeweiligen Max-Wert setzen kann. 

Bei der Gelegenheit oder später lege ich mir auch ein Feld HealthBuffer_max an da einige Spieler Ausrüstung haben die sie immer mit Puffer-LeP in einen Kampf starten lassen. 

Ich versuche das bis zur nächsten Sitzung am 30. April umzusetzen.

Gruß, Rune.
(oder sollte dies auch als Issue laufen? Bzw. wenn ich es fertig habe?)

**Comment by Gorthian** [comment]
Kannst Du ruhig auch hier lassen.
Die Idee finde ich gut, das werde ich sicherlich übernehmen.

Falls Du ein wenig Ahnung von Git und Github hast, kannst Du Dir übrigens auch gerne den passenden Branch ziehen, und Änderungen auch direkt hier mit einspielen. Ggf. muss ich mich selbst dann auch noch etwas tiefer in beides einarbeiten, aber ich freue mich über jeden der "mitbastelt".

**Comment by Gorthian** [comment]
Eine entsprechende Funktion ist in der aktuellen V3.0 enthalten
